### PR TITLE
Show correct message in information bar about mentoring in oversubscribed tracks

### PR DESCRIPTION
**Pull request by kntsoriano**
Closes https://github.com/exercism/bugs/issues/36

Screen Shot 2019-10-16 at 7 38 47 AM

### CODE REVIEW

File: test/system/my/solutions_information_bar_test.rb [hunk 1]
```diff
@@ -126,27 +126,6 @@ class My::SolutionsInformationBarTest < ApplicationSystemTestCase
     assert_selector ".notifications-bar", text: "Your exercise has been submitted successfully."
   end
 
-  test "In independent mode with slots" do
```
Comment (iHiD) [inline test/system/my/solutions_information_bar_test.rb]: Why are these two tests redundant now?

Comment (kntsoriano) [reply test/system/my/solutions_information_bar_test.rb]: I've tested these edge cases in the helper test already. I've also added a system test to request mentor feedback via the information bar.
